test/integration/wallet.test.js

```diff
@@ -192,6 +193,35 @@ describe('@integration: Wallet', () => {
       step('When I click "send button"', () => helper.clickOnElement('.send-button button'));
       step(`Then I should see text ${successMessage} in "result box message" element`, () => helper.haveTextOf('.result-box-message', successMessage));
     });
+
+    describe('Scenario: should show account initialisation option if no public key and balance is greater than 0', () => {
+      step('Given I\'m on "wallet" as "genesis" account', () => setupStep('genesis', false, false));
```

Comment:
This would be more readable as `setupStep('genesis', { isLocked = false, withPublicKey = true })` 

Or even better, instead of `withPublicKey = false`, create a new  `uninitialized account` in `/test/constants/accounts.js` for this purpose.

Reply:
👍